src/EnvelopeCollection.php

```diff
@@ -20,27 +20,19 @@
  *
  * @implements \IteratorAggregate<TestEnvelope>
  */
-final class EnvelopeCollection implements \IteratorAggregate, \Countable
+abstract class EnvelopeCollection implements \IteratorAggregate, \Countable
 {
-    private TestTransport $transport;
-
     /** @var Envelope[] */
-    private array $envelopes;
+    protected array $envelopes;
```

Comment:
for now it seems we don't need to make this protected?

Reply:
Ideed, again small error due to log time on other projects :-)